src/Neo.CLI/config.testnet.json

```diff
@@ -30,8 +30,8 @@
   "ProtocolConfiguration": {
     "Network": 894710606,
     "AddressVersion": 53,
-    "MillisecondsPerBlock": 15000,
-    "MaxTransactionsPerBlock": 5000,
+    "MillisecondsPerBlock": 3000,
+    "MaxTransactionsPerBlock": 256,
     "MemoryPoolMaxTransactions": 50000,
     "MaxTraceableBlocks": 2102400,
```

Comment:
Notice that it'd be ~73 days instead of current year. But I'd rather cut it down even more aggressively (see FS network settings).

Reply:
I agree, let's say 64 or 128?

Reply:
Opinions are welcome, your values remind me of E-letter chain. Nothing bad about that, but historic data (#2026) suggests our contracts are more used to somewhat longer history available. NeoFS networks use 17280 (3 days of 15s blocks).

Reply:
Just for the record, I have analyzed the blocks up to 6507174 height of N3 mainnet and checked all calls of native Ledger contract that affected by `MaxTraceableBlocks` setting (`getBlock`, `getTransaction`, `getTransactionHeight`, `getTransactionFromBlock`, `getTransactionSigners`, `getTransactionVMState`). Turns out that:
1. There's only 5262 call to these APIs that successfully retrieve block and check if it's traceable. It's much lower than I expected.
2. Contracts try to reach only *very recent blocks*, they don't go far than 11 blocks ago. In fact, most calls are trying to reach block accepted 1-5 blocks ago.
3. Here's some statistics on calls:

| `PersistingBlockIndex - TargetBlockIndex` | The number of occurrences |
| --- | --- |
| 1 | 921 |
| 2 | 3706 |
| 3 | 606 |
| 4 | 21 |
| 5 | 4 |
| 6 | 0 |
| 7 | 1 |
| 8 | 1 |
| 9 | 0 |
| 10 | 0 |
| 11 | 2 |

5. Just in case, here's the log I gathered to track mainnet calls: [traceable.log](https://github.com/user-attachments/files/18162926/traceable.log)

So from what I see, from the real mainnet contracts' PoW we don't even need 73 days of traceable blocks. In fact, for existing contracts we're good with 1 day. But it's too harsh, and there are other parts of the node that depends on `MaxTraceableBlocks` setting (like related RPC calls, conflicting transactions tracking, tails cutting unctionality and etc.). So I think that 3-7 days will be OK for mainnet.

Reply:
I think that, for now, let's keep the same parameter because, automatically, it will reduce proportionally.
So, from 73 days we go to 15.

Later we change in another PR.

Reply:
I see, I think it is a good reduce and it is not the purpose of this PR.
Maybe keep the current value will be adequated for now because it will also improve.

Reply:
OK, let's have #3644 for this. 3s consensus can have any MTB technically.